server.js

```diff
@@ -6577,6 +6577,48 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// JetBrains Plugins repository version integration
+camp.route(/^\/jetbrains\/plugin\/(v)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Most of the code is identical with code from downloads badge (cause we are using the same API resource). Maybe we should have one function handling this?

Reply:
So how to do that? This is the first day I use JavaScript to develop :cry:

Reply:
We can have one `camp.route` with `(d|v)`. `switch` statement should be put after checking if plugin is not found. Then handle `d` and `v` in this switch. You can use "Packagist integration" as a reference.